packages/dapp/next.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  publicRuntimeConfig: {
+    API_KEY_ALCHEMY: process.env.API_KEY_ALCHEMY,
+    API_KEY_COINMARKETCAP: process.env.API_KEY_COINMARKETCAP,
+    API_KEY_ETHERSCAN: process.env.API_KEY_ETHERSCAN,
+    MNEMONIC: process.env.MNEMONIC,
```

Comment:
Actually these env variables are being used in contract packages.  I think we don't need configure  them(API_KEY_COINMARKETCAP, API_KEY_ETHERSCAN, MNEMONIC) in dapp package

Reply:
Okay, will remove the env variables